modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/apiclient-header.mustache

```diff
@@ -52,6 +54,7 @@ public:
     static utility::string_t parameterToString(double value);
     static utility::string_t parameterToString(const utility::datetime &value);
     static utility::string_t parameterToString(bool value);
+    {{^hasModelImport}}static utility::string_t parameterToString(const ModelBase& value);{{/hasModelImport}}
```

Comment:
please also put these tags in separate lines to avoid blank lines.

Reply:
Fixed in https://github.com/OpenAPITools/openapi-generator/pull/21235/commits/6392b7bb8477f27f7cf3eaae32a5739a5f38d4d1